src/middlewares/authenticationMiddleware.ts

```diff
@@ -33,11 +33,11 @@ const isAuthenticated = async (
     next()
   } catch (error) {
     res.status(500).json({ message: 'Internal server down' })
+    console.log(error)
```

Comment:
I think you shouldn't have touched to this file as there is no new changes to be added
and console.log() is not supported by eslint try to replace it